script/release/kie-pushTag.sh

```diff
@@ -6,18 +6,23 @@ echo "target" : $target
 # clone the build-bootstrap that contains the other build scripts
 if [ "$target" == "community" ]; then
    source=kiegroup
+   remote=origin
 else
    source=jboss-integration
```

Comment:
Should we change also this? What does the `jboss-integration` mean ins this case?